boards/arm/qemu_cortex_r5/qemu_cortex_r5.yaml

```diff
@@ -0,0 +1,48 @@
+identifier: qemu_cortex_r5
+name: QEMU Emulation for Cortex-R5
+type: qemu
+simulation: qemu
+arch: arm
+toolchain:
+  - zephyr
+  - gccarmemb
+  - xtools
+ram: 64
+flash: 256
+testing:
+  default: false
+  ignore_tags:
```

Comment:
Not sure why these tags are all ignored

Reply:
Currently, the cortex R5 on the xlnx-zynmp qemu machine is not hooked up to an interrupt controller, so only the "hello world" test case will run right now.